content/en/docs/components/pipelines/run-a-pipeline.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Run a Pipeline"
+description = "Execute a pipeline on the KFP backend"
+weight = 6
++++
+
+The KFP SDK offers three ways to run a pipeline.
+
+## 1. Run from the KFP Dashboard
+The first and easiest way to run a pipeline is by submitting it via the KFP dashboard.
```

Comment:
Probably the easiest way is to run one of the preloaded pipeline samples that come with the deployment.

Reply:
Agreed, though with this page I intended to lay out general instructions for running _any_ pipeline, not just _a_ pipeline. I think it makes sense to include at least what is here, but do you think it's worth including how to run one of the preloaded pipelines also? It's slightly captured from step 4 and 5, but not explicitly. WDYT?

Reply:
Make sense, preloaded samples are special, we can skip it in this context.